universum/modules/structure_handler.py

```diff
@@ -51,23 +51,23 @@ class Block:
     True
     """
 
-    def __init__(self, name: str, parent: 'Block' = None):
-        self.name = name
-        self.status = "Success"
-        self.children = []
+    def __init__(self, name: str, parent: Optional[Block] = None):
```

Comment:
In order to use it with PyCharm, the name of the not yet defined class has to be expressed as a string literal: 'Block'

Reply:
The manual: https://www.python.org/dev/peps/pep-0484/#forward-references

Reply:
ok, will fix